src/api/app/controllers/webui/project_controller.rb

```diff
@@ -56,24 +56,24 @@ def index
   end
 
   def autocomplete_projects
-    render json: Project.autocomplete(params[:term]).not_maintenance_incident.order(:name).pluck(:name)
+    render json: Project.autocomplete(params[:term]).not_maintenance_incident.order(:name).limit(50).pluck(:name)
```

Comment:
I think that limit the results to 50 is not intuitive. If the user is looking for a package that is not in this 50, this is more confusing that helping. I would better not show anything if there are more than 50, than just show the first 50.

Reply:
It's an autocomplete. I doubt users get confused by getting a sub-list of possible matches. And I also doubt users want to scroll through hundreds of results.

And like @coolo I was confused when I entered 'hom' in the old search and didn't got any results.